packages/x-adapter/.eslintrc.js

```diff
@@ -1,4 +1,5 @@
 module.exports = {
+  extends: ['plugin:@empathyco/x/recommended'],
```

Comment:
Why did you add that config, it should inherit the root `.eslintrc.js` file, which already adds the recommended eslint-plugin-x config

Reply:
My bad, totally forgot it was already set up there.